### DISCUSSION TITLE
Cannot import playlist from Freetube using "Export playlist for Older version of Freetube"

### DISCUSSION THREAD

**Discussion by jeromebruzaud** [libre-tube/LibreTube#6640] [General]
### Steps to reproduce

1. Activate FreeTube options "Export Playslits for Older Freetube Versions"
This options creates a unique "Favorites" playlist and put every videos from FreeTube playlists inside it.
2. Export playlists from FreeTube
3. Transfert playlist file to my Android phone
4. Open LibreTube
5. Delete all existing playlists
6. Import playlist file by selecting "FreeTube" format

### Expected behavior

The file is loaded, the playlists are created and populated. I get the same playlist as on my FreeTube application on my PC.

### Actual behavior

The file is loaded, the "Favorites" playlist is created, but it is not populated.

### LibreTube version

0.25.2 (57)

### Android version

GrapheneOS based on Android 14 2024101200

### Other details

FreeTube version is v0.21.3 Beta
[freetube-playlists-as-single-favorites.zip](https://github.com/user-attachments/files/17396321/freetube-playlists-as-single-favorites.zip)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Is it possible that you're using LibreTube with a Piped account? As described in https://github.com/libre-tube/LibreTube/issues/6389, playlists using Piped accounts are not functional on public instances, and thus you should not use an account if you want to use playlists (or use a self-hosted instance that's not blocked by YouTube).
